hyperapp.d.ts

```diff
@@ -17,10 +17,19 @@ export interface VNode<Attributes = {}> {
  *
  * @memberOf [VDOM]
  */
-export interface Component<Attributes = {}, State = {}, Actions = {}> {
-  (attributes: Attributes, children: Array<VNode | string>):
-    | VNode<Attributes>
-    | View<State, Actions>
+export interface Component<Attributes = {}> {
```

Comment:
So here I split out lazy component from component. I'm not sure of maintainers view but normal components should only be passed attributes (aka properties) as describe in the readme.

To my mind Lazy Component should really be called Context Components as they let you pass the global context down directly (like React context). This appears to be the main attribute. Lazy to me means leaving till last possible minute and has always confused me

Reply:
Why does Component remain interface? In hyperapp, Component is just a function that returns VDOM Node as well as View, so I think that Component should be of type as well as View.

Reply:
No argument from me - I just split it out and duplicated :)

Reply:
OK :)